### PR TITLE
Elemental2

### PR DESCRIPTION
**Pull request by vegegoku**
- modify the demo to use the elemental2 version of the webcomponents.
- Replace JsArray with elemental2.core.Array
- Update GWT version 2.8.0 -> 2.8.1
- Update gwt-maven-plugin 1.0-rc-6 -> 1.0-rc-8

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (manolo) [inline pom.xml]: Don't commit this file, it's generated by gwt-api-generator

File: demo/package.json [hunk 2]
```diff
@@ -5,6 +5,6 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/vaadin/gwt-polymer-elements.git",
   "dependencies": {
-    "gwt-api-generator": "vaadin/gwt-api-generator#master"
+    "gwt-api-generator": "vaadin/gwt-api-generator#elemental2"
```
Comment (manolo) [inline demo/package.json:8]: This should be changed before merging. Leave it as it is for now.

**Review by manolo — CHANGES_REQUESTED** [review]

File: demo/package.json (hunk 2 above)
Comment (vegegoku) [reply demo/package.json:8]: ok then i will change it back to elemental2
and the pom file should i change how it is generated by the gwt-api-generator?

Comment (vegegoku) [reply demo/package.json:8]: this one is done

File: pom.xml (hunk 1 above)
Comment (vegegoku) [reply pom.xml]: i added the changes to the gwt-api-generator to generate the pom with the required changes

Comment (vegegoku) [reply pom.xml]: pom.xml now is removed

**Review by manolo — APPROVED** [review]

**Pull request merged**